rollemup-launcher.sh

```diff
@@ -0,0 +1,30 @@
+# Save variable files in user home directory.
+data_dir=${XDG_DATA_HOME:-$HOME/.local/share}/rollemup
+mkdir -p $data_dir/Games
+touch $data_dir/Rollemup.ini
+
+# CWD must be in the directory containing game resources to make it working.
+# Also, CWD must be writeable for highscores and configuration.
+mkdir /tmp/rollemup-runtime
```

Comment:
Given $XDG_DATA_HOME is writable, why not create fake directory structure there?

Reply:
I don't think it particularly matters as these are just symlinks but up to you.